docs/user/supported_devices.rst

```diff
@@ -275,6 +279,7 @@ ramips-mt7621
 * ASUS
 
   - RT-AC57U
+  
```

Comment:
why did you add this empty line?

Reply:
I think accidentally. In a first version, I've put the RT-AC51U under the RT-AC57U. Then I obviously deleted only the content, not the line.

Reply:
I removed the unnecessary empty line.